_includes/specials/policy/en/08-interface.md

```diff
@@ -0,0 +1,119 @@
+So far in this series, we have explored the [motivations][policy01]
+and challenges associated with decentralized transaction relay,
+leading to a [local][policy05] and [global][policy07] need for
+transaction validation rules more restrictive than consensus. Since
+transaction relay policy changes to Bitcoin Core can impact whether an
+application’s transactions relay, they require socialization with the
+wider Bitcoin community prior to consideration. Similarly,
+applications and second layer protocols that utilize transaction relay
+must be designed with policy rules in mind to avoid creating
+transactions that are rejected.
+
+Contracting protocols are even more intimately dependent on policies
+related to prioritization because enforceability on-chain depends on
+being able to get transactions confirmed quickly. In adversarial
+environments, cheating counterparties may have an interest in delaying
+a transaction’s confirmation, so we must also think about how quirks
+in the transaction relay policy interface can be used against a user.
+
+Lightning Network transactions adhere to the standardness rules
+mentioned in earlier posts. The peer-to-peer protocol specifies a
+`dust_limit_satoshis` and `max_accepted_htlcs` in its
+`open_channel` message to specify a dust threshold and maximum number
+of in-flight HTLCs. Since a transaction containing an output with a
+value lower than the dust threshold would not relay due to nodes’ dust
+limits, those payments are considered “not enforceable on-chain” and
+trimmed from commitment transactions. The maximum number of in-flight
+HTLCs cannot exceed 483 per channel, as a commitment transaction with
+more than 483 HTLC outputs would exceed the maximum standard
```

Comment:
> a commitment transaction with more than 483 HTLC outputs would exceed the maximum standard size

I don't believe this is correct.  First, the LN rule is 483 outputs **per party**, so it's 966 outputs plus 2 more outputs for the each party's base commitment outputs.


Second P2WSH 1-input, 968-outputs is  just ~42,000 vbytes.  The reason for the limitation per [BOLT5](https://github.com/lightning/bolts/blob/master/05-onchain.md#rationale-6) is (emphasis mine), "A single transaction that **resolves** all the outputs will be under the standard size limit because of the 483 HTLC-per-party limit".

LN doesn't require all the HTLCs to be resolved in a single transaction---it's just more efficient to do it that way than in individual transactions, and I suppose it also simplifies implementation.  Of note, many (most?) LN implementations currently soft-limit themselves by default to a lower number of per-party pending HTLCs than 483 to reduce their exposure to certain attacks, which I believe is why nobody is pushing to increase the 483 HTLC limit.

Given this section has several other examples of the influence of policy on contracting protocol rules, I think the simplest thing would be to drop this sentence.

Reply:
Thanks for catching this! I had a pretty surface-level understanding of this and learned something new today :D